setup.py

```diff
@@ -35,7 +35,7 @@
           url=URL,
           version=VERSION,
           download_url=DOWNLOAD_URL,
-          long_description=open('README.rst').read(),
+          long_description=open('README.md').read(),
           long_description_content_type='text/x-rst',
```

Comment:
this would have to be adjusted for markdown, or does text/x-rst now work for both markdown and rst?

Reply:
Whoops. Yes probably needs adjustment.

Reply:
Done in 57c8733